packages/thirdweb/src/react/web/ui/ConnectWallet/screens/Buy/BuyScreen.tsx

```diff
@@ -134,33 +141,41 @@ function BuyScreenContent(props: BuyScreenContentProps) {
     id: "main",
   });
 
+  // --------------------------------------------------------------------------
```

Comment:
This block of code is not net new - copied over directly from `useUISelectionStates`